web/impact/impact/models/utils.py

```diff
@@ -8,7 +8,7 @@
 
 
 def is_managed(db_table):
-    return False
+    return True
```

Comment:
I don't like the fact that the boolean flag is configured in settings.py in one project and is set literally in the other.